pyEpiabm/pyEpiabm/utility/distance_metrics.py

```diff
@@ -0,0 +1,30 @@
+#
+# Calculates distance metrics between spatial points
+#
+
+import typing
+import numpy as np
+
+
+class DistanceFunctions:
+    """Class which contains multiple distance functions
+    for use when considering spatial infections, either
+    inter-place or inter-cell.
+    """
```

Comment:
What if we just create a staticmethod dist here which then uses dist_euclid. Will make it easier to change method in the future.